### DISCUSSION TITLE
Pamac won't open, only command line interface

### DISCUSSION THREAD

**Discussion by GladiatorOnWeekends** [sileshn/ManjaroWSL2#22] [Q&A] [answered]
Hi, I'm new to Linux. New to computers as a whole in fact it seems. i though i knew how it all worked, comfortably on my Windows OS, but now that i know how much things you have to enable yourself, i understand i did not understood at all how computers work. Anyways, i entered this command : 
$ pamac-manager 
and received this : 
(pamac-manager:86): Gtk-WARNING **: 22:00:06.157: Failed to open display

What am I supposed to do ? Should I install it again ? Do I need a whole KDE, GNOME or LXQt just to open that pacman graphical interface ? Talking, about desktop environments, is LXQt a good option ? Is quite easy on ressources, and on the official desktop environment website you link in the README, Plasma KDE 6 has a side note reading : 
Warning
The various components for Plasma which is available using the internet, should be used with care. Theming on Plasma is not just a bunch of icons and colors - it is based on QML, the Qt model language which is specific for the used version of Qt.
What does this mean in normal-people english. Should I worry about it ?

**Comment by sileshn** [comment]
wsl2 is meant to be run as cli, meaning its run using the command prompt. If you need GUI support, you need to perform additional steps to get it working, though success is never guaranteed.

You can try [these](https://github.com/sileshn/ManjaroWSL2?tab=readme-ov-file#how-to-setup-a-desktop-environment) instructions to install a Plasma desktop on ManjaroWSL2. Xfce would be the best to run. There are numerous guides on the web with the necessary instructions.

**Reply by GladiatorOnWeekends** [reply]
Ok, ill try that, thanks for such a fast answer on an old project. You really are committed to the community. I have installed gtk for pamac through the command : 
user $ sudo pacman -Syu pamac-gtk
How should I uninstall it now ? Will this also uninstall my pamac-cli ?

**Reply by sileshn** [reply, marked as answer]
sudo pacman -R pamac-gtk. It will not remove pamac-cli
